create_infotexts.py

```diff
@@ -135,6 +135,25 @@ def create_people_mapping_wikitable(people_mapping):
 
     return table
 
+
+def create_smvk_mm_link(item):
+    """Populates template SMVK-MM-link and appends to dictionary."""
```

Comment:
Rather than concattenating strings this function might be easier to understand if it was something like
`smvk_link = "{{{{SMVK-MM-link|{postnr}|{fotonr}}}}}".format(postnr=row["Postnummer"], fotonr=row["Fotonummer"])`

Note that `{{` becomes just `{` if you use format. Hence the need for four `{` to create the template markup.